src/Shared/HttpSys/RequestProcessing/NativeRequestContext.cs

```diff
@@ -478,5 +480,43 @@ private uint GetChunksHelper(ref int dataChunkIndex, ref uint dataChunkOffset, b
             }
             return dataRead;
         }
+
+        internal IReadOnlyDictionary<int, ReadOnlyMemory<byte>> GetRequestInfo()
+        {
+            if (_permanentlyPinned)
```

Comment:
How frequently do both codepaths get hit here? What is the default setting for pinning?

Reply:
_permanentlyPinned is for IIS, HttpSys always uses the other one.

Reply:
Oh yeah... forgot I added that XD